apps/comments/lib/Controller/Notifications.php

```diff
@@ -43,31 +43,17 @@
  * @package OCA\Comments\Controller
  */
 class Notifications extends Controller {
-	/** @var IRootFolder  */
-	protected $rootFolder;
 
-	/** @var ICommentsManager  */
-	protected $commentsManager;
-
-	/** @var IURLGenerator  */
-	protected $urlGenerator;
-
-	/** @var IManager  */
-	protected $notificationManager;
-
-	/** @var IUserSession  */
-	protected $userSession;
+	protected IRootFolder $rootFolder;
+	protected ICommentsManager $commentsManager;
+	protected IURLGenerator $urlGenerator;
+	protected IManager $notificationManager;
+	protected IUserSession $userSession;
 
 	/**
 	 * Notifications constructor.
 	 *
 	 * @param string $appName
-	 * @param IRequest $request
-	 * @param ICommentsManager $commentsManager
-	 * @param IRootFolder $rootFolder
-	 * @param IURLGenerator $urlGenerator
-	 * @param IManager $notificationManager
-	 * @param IUserSession $userSession
 	 */
 	public function __construct(
 		$appName,
```

Comment:
Any chance of strong typing $appName as string?
```suggestion
		string $appName,
```